packages/zendesk-adapter/src/filters/unordered_lists.ts

```diff
@@ -281,9 +281,27 @@ const orderMacroIds = (workspace: InstanceElement): void => {
     log.trace(`macro_ids in workspace ${workspace.elemID.getFullName()} is not an array`)
   }
 }
+
+type MacroRestriction = {
+  ids?: ReferenceExpression[]
+}
+
+const sortSelectedMacros = (selectedMacros: { restriction?: MacroRestriction }[]): void =>
+  selectedMacros.forEach(macro => {
+    if (macro.restriction !== undefined && _.isObject(macro.restriction) && 'ids' in macro.restriction) {
+      const restrictionIds = _.get(macro, 'restriction.ids')
+      if (_.isArray(restrictionIds) && restrictionIds.every(isReferenceExpression)) {
+        macro.restriction.ids = _.sortBy(restrictionIds, ref => ref.elemID.getFullName())
+      } else {
+        log.warn(`could not sort restriction ids for restriction ${macro.restriction}`)
```

Comment:
For easier debugging
```suggestion
        log.warn(`could not sort restriction ids for restriction ${macro.restriction}`)
        log.trace(`restriction ids are: ${inspectValue(restrictionIds, { maxArrayLength: null })}`)
```